server/blob/blob-union/src/test/java/org/apache/james/blob/union/HybridBlobStoreTest.java

```diff
@@ -276,6 +285,58 @@ void saveInputStreamShouldRelyOnLowCostWhenSizeBasedAndBig() {
         }
     }
 
+    @Nested
+    class StoragePolicyWithHybridDuplicateWriteTests {
+
+        @Test
+        void saveShouldRelyOnPerformingAndDuplicateWriteWhenPerforming() {
```

Comment:
Please also do this test with large data (just to be sure we do handle duplicates writes well: I suspect the stream content not being handled well in that very case!)

For this one and all the tests below.

Reply:
> I suspect the stream content not being handled well in that very case!

I dont actually understand it.
BTW `large data case` are tested with `hybridBlobStore` (default DuplicateWrite = `false`)